docs/guides/Azure-Installation.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Installing JTL Reporter on Azure container instances 
+---
+
+### Prerequisites
+* Azure Account 
+* Latest version of [PSQLODBC](https://www.postgresql.org/ftp/odbc/versions/msi/) to run psql commands.
+
+### Preparation
+
+#### Create Azure Resources 
+you need to create the following azure resources before u begin:
+* Azure Resorce group.
+* Azure storage account with 2 file shares (for mongo and Postgres databases).
+* Azure storage account access key.
+
+#### Instalation steps: 
+1. Create Azure resource group 
+2. Create azure storage account with 2 file shares.
+3. Get the storage account access key.
+4. clone the [git repo](https://github.com/esahanhunter/AzureJTLReporter).

```

Comment:
Please make the first letter capital

Reply:
Would you be willing to move the deployment scripts into https://github.com/ludeknovy/jtl-reporter ?

Reply:
updated the typos and have created the PR to push the scripts. :)